src/demo/containers/playground/ToastDemo.md

```diff
@@ -0,0 +1,3 @@
+```js
+<ToastDemo />
```

Comment:
Could you also include an example code snippet here that uses displayToast() (probably would need to make it globally available in the StyleGuidist setup)?

Reply:
Sure, I'll put the demo code here instead of wrapping it in a component